akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.pubsub
+
+import akka.actor.testkit.typed.scaladsl.LoggingTestKit
+import akka.actor.testkit.typed.scaladsl.LogCapturing
+import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import org.scalatest.wordspec.AnyWordSpecLike
+import scala.concurrent.duration._
+
+class LocalPubSubSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike with LogCapturing {
+
+  "A pub-sub topic running locally" must {
+
+    "publish to all local subscriber actors of a topic" in {
+      val fruitTopic =
+        LoggingTestKit.debug("Topic list updated").expect {
+          testKit.spawn(Topic[String]("fruit"))
+        }
+
+      try {
+        val probe1 = testKit.createTestProbe[String]
+        val probe2 = testKit.createTestProbe[String]
+        val probe3 = testKit.createTestProbe[String]
+
+        LoggingTestKit.debug("Topic list updated").expect {
```

Comment:
Maybe different lots for when we've registered out selves?

Reply:
We don't know it's the same logic whenever the set of topic instances has changed